xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -910,6 +911,7 @@ In application code, this field is most commonly used to safely represent a <xre
         <returns>
           <see langword="true" /> if the value of <paramref name="left" /> is the same as the value of <paramref name="right" />; otherwise, <see langword="false" />. If <paramref name="right" /> is <see langword="null" />, the method returns <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
+        <inheritdoc cref="M:Microsoft.Extensions.Primitives.StringValues.Equals(Microsoft.Extensions.Primitives.StringValues,System.String)" />
```

Comment:
The tool imported this inheritdoc correctly, but mdoc complains it is wrong.
https://github.com/dotnet/runtime/blob/38ba41ef6f29cb7a3ca50ea93ba47abb785e2654/src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs#L632

I guess it's not possible to inherit from a sibling. I'll remove it. It's fully documented anyway.

```suggestion
```